apollo-router/src/response.rs

```diff
@@ -128,6 +134,24 @@ impl Response {
                 service: service_name.to_string(),
                 reason: err.to_string(),
             })?;
+        let incremental =
+            extract_key_value_from_object!(object, "incremental", Value::Array(a) => a).map_err(
```

Comment:
is incremental mandatory? will this work regardless of the experimental defer support flag value ?

Reply:
it is not mandatory. The `from_bytes` method is used to parse a response from a subgraph, and we don't support pass-through defer yet, so this won't do anything right now